apps/admin-x-settings/src/components/settings/email/newsletters/NewsletterPreviewContent.tsx

```diff
@@ -286,8 +287,8 @@ const NewsletterPreviewContent: React.FC<{
                                                 titleFontWeight === 'semibold' && 'font-semibold',
                                                 titleFontWeight === 'bold' && 'font-bold'
                                             )}
-                                            style={{color: sectionTitleColor}}>When Convenience Comes at a Cost</h3>
-                                        <p className="mb-6" style={{color: textColor}}>So, what’s the future of food culture in an on-demand world? Can these platforms adapt to better support small businesses and workers? Or will we wake up one day to find that the places we once loved have vanished?</p>
+                                            style={{color: titleColor}}>When Convenience Comes at a Cost</h3>
+                                        <p className="mb-6" style={{color: textColor}}>So, what&apos;s the future of food culture in an on-demand world? Can these platforms adapt to better support small businesses and workers? Or will we wake up one day to find that the places we once loved have vanished?</p>
```

Comment:
Would be good to keep unrelated fixes in separate PRs, keeps history clearer and makes any potential rollbacks less risky 🙂

Reply:
Think this is a bad conflict merge. Fixed it. :)